### PR TITLE
Getting Started with Django Rest Framework and AngularJS

### PR DESCRIPTION
**Pull request by KetsuN**
Think this would be a great blog post. @Jwpe gave a great answer on reddit[1], and the recommended article[2] seemed lacking in in-depth content on the subject...

[1]http://www.reddit.com/r/django/comments/26jpia/folder_structure_for_python_django_rest_framework/
[2]http://blog.kevinastone.com/getting-started-with-django-rest-framework-and-angularjs.html

### CODE REVIEW

**Comment by joshfinnie** [comment]
@KetsuN how's this coming? Any idea when Part 1 would be ready for release?

**Comment by KetsuN** [comment]
@joshfinnie Should be ready for review.  This would be awesome to post before the end of this cycle is it looks ready.

**Comment by joshfinnie** [comment]
@KetsuN this looks great. Is the idea to run the angular project completely separated from the django project?

File: content/getting-started-drf-angularjs-part-1.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+├── drf_sample
+│   ├── __init__.py
+│   ├── settings.py
+│   ├── urls.py
+│   └── wsgi.py
+└── manage.py
+```
+Cool, but what do these files actually do?
+
+|File Name|Description|
```
Comment (joshfinnie) [inline content/getting-started-drf-angularjs-part-1.md]: Not sure how I feel about the table, for some reason it looks a bit funky when generated through pelican:

screen shot 2015-11-10 at 2 08 11 pm

Thoughts on using [HTML definition lists](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)?

**Comment by joshfinnie** [comment]
@fheisler wanna read through this for grammar/spelling? you are much better at that than I...

**Comment by KetsuN** [comment]
@joshfinnie Thanks for looking through this.  Yeah, the end the goal is to run them separately.  I feel like it has more bearing in our moving forward to separate services.  I'll look into the definition lists!

**Comment by KetsuN** [comment]
HTML definitions and a little custom styling.  I think this looks much better now.

image

**Comment by joshfinnie** [comment]
@KetsuN yes, I do too...

File: content/getting-started-drf-angularjs-part-1.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+│   ├── __init__.py
+│   ├── settings.py
+│   ├── urls.py
+│   └── wsgi.py
+└── manage.py
+```
+Cool, but what do these files actually do?
+
+<dl>
+  <dt>settings.py</dt>
```
Comment (joshfinnie) [inline content/getting-started-drf-angularjs-part-1.md]: If you want the markdown `<backtick>` to happen here in HTML, you'll need to surround `settings.py` with `<code>`:

```
<code>settings.py</code>
```

Only add that if you want to...

**Comment by joshfinnie** [comment]
@KetsuN made one more comment...

**Comment by KetsuN** [comment]
@joshfinnie I'm not too worried about it in that chart.  It'd not truly code, so it doesn't need the block look.

**Comment by joshfinnie** [comment]
Awesome. Then I think this is good to go. We can release it tomorrow! Woot!!

**Comment by fheisler** [comment]
I believe! Will read over for grammar etc if I get a chance later this week, but would rather get it posted period :)

**Pull request merged**